dev/io.openliberty.xmlBinding.4.0.internal.tools/bnd.bnd

```diff
@@ -0,0 +1,112 @@
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props, \
+          ~../cnf/resources/bnd/app-resources.bnd
+
+bVersion=1.0.${libertyBundleMicroVersion}
+bSymbolicName=io.openliberty.xmlBinding.4.0.internal.tools
+
+Bundle-Name: io.openliberty.xmlBinding.4.0.internal.tools
+Bundle-SymbolicName: io.openliberty.xmlBinding.4.0.internal.tools; singleton:=true
+    
+# The SPI definition under META-INF/services
+# Do not publish RI javax.xml.bind.JAXBContext
+app-resources= \
+  META-INF/services/com.sun.tools.xjc.Plugin |\
+  META-INF/services/org.relaxng.datatype.DatatypeLibraryFactory | \
+  META-INF/services/org.iso_relax.verifier.VerifierFactoryLoader
+
+Service-Component: \
+  io.openliberty.jaxb.internal.tools_4_0_${replace;${bVersion};\\.;_}.ResourceProvider; \
+    implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
+    provide:=com.ibm.wsspi.classloading.ResourceProvider; \
+    configuration-policy:=ignore; \
+    properties:= "resources=${app-resources}",\
+  org.glassfish.jaxb.runtime.v2.JAXBContextFactory; \
+    implementation:=org.glassfish.jaxb.runtime.v2.JAXBContextFactory; \
+    provide:=jakarta.xml.bind.JAXBContextFactory; \
+    configuration-policy:=ignore
+
+
+Import-Package: \
+  jakarta.activation.*,\
+  jakarta.xml.bind.*; version="[4.0,5.0]", \
+  com.sun.xml.tools.*, \
+  com.ibm.wsspi.classloading, \
+  org.apache.xml.*; resolution:=optional, \
+  org.w3c.dom.*, \
+  !javax.activation, \
+  javax.xml.catalog;resolution:=optional, \
+  javax.xml.transform.*, \
+  javax.xml.parsers.*, \
+  javax.xml.namespace.*, \
+  javax.xml.datatype.*, \
+  javax.xml.validation.*, \
+  javax.xml.xpath.*, \
+  org.xml.sax.*, \
+  !com.sun.xml.internal.bind.v2, \
+  !com.sun.xml.bind.v2, \
+  !org.apache.tools*, \
+  !org.apache.xerces*, \
+  !org.jvnet*, \
+  !com.sun.mirror.*, \
+  com.sun.codemodel.*,\
+  !com.sun.source.tree.*, \
+  !com.sun.source.util.*, \
+  !com.sun.xml.dtdparser.*, \
+  !com.sun.xml.fastinfoset.sax.*, \
+  !com.sun.xml.fastinfoset.stax.*, \
+  !sun.nio.cs*, \
+  !com.sun.org.apache.xml.internal.resolver, \
+  !com.sun.org.apache.xml.internal.resolver.tools, \
+  org.glassfish.hk2.*, \
+  *
+    
+Export-Package: \
+  org.glassfish.jaxb.*;version="4.0";thread-context=true
+
+Include-Resource: \
+  @${repo;org.glassfish.jaxb:codemodel;4.0.0}!/!(META-INF/maven/*|module-info.class), \
+  @${repo;org.glassfish.jaxb:jaxb-core;4.0.0}!/!(META-INF/maven/*|module-info.class), \
+  @${repo;org.glassfish.jaxb:jaxb-jxc;4.0.0}!/!(META-INF/maven/*|module-info.class), \
+  @${repo;org.glassfish.jaxb:jaxb-runtime;4.0.0}!/!(META-INF/maven/*|module-info.class), \
+  @${repo;org.glassfish.jaxb:jaxb-xjc;4.0.0}!/!(META-INF/maven/*|module-info.class), \
```

Comment:
Any value in having xjc here? Are you changing things at all to making a XMLBinding 4.0 script?